package.json

```
@@ -42,8 +42,8 @@
     "@glif/filecoin-address": "^2.0.1",
     "@influxdata/influxdb-client": "^1.33.0",
     "@sentry/node": "^7.29.0",
```

```suggestion
    "@sentry/node": "^7.30.0",
```

Let's upgrade all Sentry deps together